frontends/election-manager/src/config/features.ts

```diff
@@ -0,0 +1,57 @@
+import { unsafeParse } from '@votingworks/types';
+import { asBoolean } from '@votingworks/utils';
+import { ConverterClientType, ConverterClientTypeSchema } from './types';
+
+/**
+ * Determines whether authentication is enabled.
+ *
+ * To disable authentication, add this line to `frontends/election-manager/.env.local`:
+ *
+ *     REACT_APP_VX_BYPASS_AUTHENTICATION=true
+ *
+ * To enable it, remove the line or comment it out. Restarting the server is required.
+ *
+ * @see https://create-react-app.dev/docs/adding-custom-environment-variables/
+ */
+export function isAuthenticationEnabled(): boolean {
+  return (
+    process.env.NODE_ENV !== 'development' ||
+    !asBoolean(process.env.REACT_APP_VX_BYPASS_AUTHENTICATION)
+  );
+}
+
+/**
+ * Determines whether write-in adjudication is enabled.
+ *
+ * To enable write-in adjudication, add this line to `frontends/election-manager/.env.local`:
+ *
+ *     REACT_APP_VX_WRITE_IN_ADJUDICATION=true
```

Comment:
just noticed that this comment doesn't have the "ENABLE" thats in the actual code check